### DISCUSSION TITLE
copypaste later added pfields

### DISCUSSION THREAD

**Discussion by tjingboem** [kunstmusik/blue#635] [Ideas]
i decided i needed to add another pfield to an Object, in this case the PianoRoll. So this would be pfield 7, named XVOL here.
On the Timeline i have a few dozen Objects for this particular instrument.
![Screenshot from 2021-11-08 12-44-44](https://user-images.githubusercontent.com/6670911/140736764-4fb27110-2aa2-4a94-b950-5ee486f48851.png)

An option to be able to add this pfield to all other pfields would save me a lot of time. In this case a PianoRoll is used, and i am not sure how this new pfield should be copied over to other than PianoRolls for the same instruments..but that is something to discuss and think about?

**Comment by kunstmusik** [comment]
Interesting use case, certainly can see how it would be useful. I wonder if there might be a way to develop some kind of query tool that allowed selecting by parameters, then offering certain bulk operations depending on what types are selected. Could be more of a generic tool then to do operations in bulk. Thoughts?

**Comment by tjingboem** [comment]
Bulk operations depending on the same types selected should be feasible. If it is of another type, perhaps some utility where strings could be filled in is the only way.
A selector for Object Type could show the options for that type only.
